protocol/communities/community.go

```diff
@@ -723,7 +718,23 @@ func (o *Community) RemoveUserFromOrg(pk *ecdsa.PublicKey) (*protobuf.CommunityD
 	}
 
 	o.increaseClock()
+}
+
+func (o *Community) RemoveOurselvesFromOrg(pk *ecdsa.PublicKey) {
+	o.mutex.Lock()
+	defer o.mutex.Unlock()
+	o.removeMemberFromOrg(pk)
+}
```

Comment:
Why exactly is this called `RemoveOurselvesFromOrg` ?
This seems a pretty generic function. In fact it seems to be just an exposed API for `removeMemberFromOrg`.

Reply:
If you compare `RemoveOurselvesFromOrg` with `RemoveUserFromOrg` you will see they only differ with one check. If we remove ourselves from org we don't need to check if we are admins.

The other option was to have one function with an additional boolean parameter:
`RemoveUserFromOrg(pk *ecdsa.PublicKey, requireAdmin bool)`

Since boolean arguments are bad for code readability (see example below), I decided to go with the first one.

```
func (a *Api) LeaveCommunity() {
    community.RemoveOurselvesFromOrg(m.identity.pubKey)
}
func (a *Api) RemoveUserFromOrg(pubKey *PubKey) {
    community.RemoveUserFromOrg(pubKey)
}
```
vs
```
func (a *Api) LeaveCommunity() {
    community.RemoveUserFromOrg(m.identity.pubKey, false)
}
func (a *Api) RemoveUserFromOrg(pubKey *PubKey) {
    community.RemoveUserFromOrg(pubKey, true)
}
```